calibr8/core.py

```diff
@@ -525,16 +531,17 @@ def loglikelihood(
                 if not name:
                     raise ValueError("A `name` must be specified for the PyMC likelihood.")
                 rv = self.pymc_dist(name, **self.to_pymc(*params), observed=y, **dist_kwargs or {})
+                return pmodel.logp([rv], sum=True)
             else:
                 rv = self.pymc_dist.dist(**self.to_pymc(*params), **dist_kwargs or {})
-            # The API to get log-likelihood tensors differs between PyMC versions
-            if pm.__version__[0] == "3":
-                if isinstance(rv, pm.model.ObservedRV):
-                    return rv.logpt.sum()
-                elif isinstance(rv, pm.Distribution):
-                    return rv.logp(y).sum()
-            else:
-                return pm.joint_logpt(rv, y, sum=True)
+                # The out-of-modelcontext API for log-likelihoods differs between PyMC v4 and v5
```

Comment:
```suggestion
                # The out-of-model context API for log-likelihoods differs between PyMC v4 and v5
```

Something seems off in the spelling here.

Reply:
The spelling aligns with `pm.modelcontext`..